packages/sdk/src/modules/DatasourceUtility.ts

```diff
@@ -0,0 +1,48 @@
+import type { GetUnspentsResponse } from "../api/types";
+import type { Inscription } from "../inscription/types";
+import type { UTXO } from "../transactions/types";
+import { UNSTABLE_decodeObject } from "../utils";
+
+interface SegregateUTXOsBySpendStatusArgOptions {
+  utxos: UTXO[];
+}
+
+class DatasourceUtility {
+  static transformInscriptions(inscriptions?: Inscription[]) {
```

Comment:
i think this function can be better named to specify explicitly that it decodes the `meta`